packages/administration/src/Component/Datagrid/DatagridActions.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\AdministrationBundle\Component\Datagrid;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use InvalidArgumentException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @phpstan-type ActionOptions array{
+ *     label?: string,
+ *     icon?: string,
+ *     routeName?: string,
+ *     additionalParameters?: array,
+ *     confirmMessage?: string,
+ * }
+ */
+final class DatagridActions
```

Comment:
maybe this should be named rather `DatagridRowActions` as in the future we might want to add actions for the grid as a whole (or is it already there? I am not sure)

Reply:
I refactored Datagrid actions to DatagridRowActions and they are now defined by same AbstractAction class as CrudActions :)